src/emuseum.js

```diff
@@ -2,6 +2,39 @@ module.exports = ({ emuseumKey }) => {
   const fetch = require('node-fetch')
   const { MISSING_EMUSEUM_KEY } = require('./errors.js')
 
+  const parseRawObject = (raw) => {
+    let {
+      id: { value: id },
+      title: { value: title },
+      medium,
+      classification,
+      creditline,
+      dimensions,
+      primaryMaker,
+      primaryMedia,
+      people
+    } = raw
+
```

Comment:
for consistency sake we can make id and title like the lines below

```js
id = id && id.value
title = title && title.value
```
we might want an `|| null`, or a more generic / strict filter in the future

Reply:
cool will do